### ISSUE TITLE
Setup test performance environment on AWS with A/B testing capabilities

### ISSUE THREAD

**Issue by fuziontech** [PostHog/posthog#1282]
We have a great opportunity to test out performance tweaks and ensure our customers have a good experience on Posthog.

Use TeeProxy to reverse proxy events to multiple backends with only one being the primary.

Things to test:

- [x] Aurora vs PG
- [ ] Kinesis for event pipelining
- [x] Changes in backend logic (view this as a staging environment)

Things we will need to do once we have this setup:
- [x] Instrument all the metrics
- [ ] Setup CD to all (n number) environments

**Comment by fuziontech** [comment]
Quick update here: Environment is setup but having issues with minting a new cert for TLS. Going to debug tomorrow.

This is what the setup looks like currently:
![openrent a_b infra (1)](https://user-images.githubusercontent.com/391319/88617491-a81fc380-d04b-11ea-95c6-bc90baf6ff84.png)

**Comment by timgl** [comment]
Doesn't Cloudflare automatically give you a TLS cert?

I also don't understand the schema. don't we want 100% of traffic to go to both tests?

**Comment by fuziontech** [comment]
Cloudflare does give you TLS termination in proxy mode but you still need to setup TLS to the final destination. In this case I did end up using a cloudflare signed cert but I had to track down the cloudflare chain cert since ELB won't let you use a self signed cert.

The two environments are now deployed and receiving 100% of production event traffic. We can use the `B` environment for testing out any changes we want to make.

Also setup is the new [metrics.posthog.com](metrics.posthog.com) statsd stack.
